### DISCUSSION TITLE
Redirects and .netrc

### DISCUSSION THREAD

**Discussion by jjagielka** [encode/httpx#2087] [Potential Issue] [answered]
`httpx` uses `.netrc` credentials for automatic login when http call is triggered, but it does not apply `.netrc` credentials when doing redirecting (`follow_redirects=True`)

Site 1 -> Site 2 -> Site 3
If the authorization is required on Site2, `follow_redirects` won't work even if `.netrc` contains an entry for that site.

`requests` does apply `.netrc` for traversed sites.

Is that a bug?

**Comment by lovelydinosaur** [comment, marked as answer]
Sure sounds like one, yup. Want to open an issue for it?

**Reply by lovelydinosaur** [reply]
Opened as https://github.com/encode/httpx/issues/2088
